tests/interfaces/records/handlers/records-read.spec.ts

```diff
@@ -135,6 +138,189 @@ describe('RecordsReadHandler.handle()', () => {
       expect(Comparer.byteArraysEqual(dataFetched, dataBytes!)).to.be.true;
     });
 
+    describe('protocol based reads', () => {
+      it('should allow read with allow-anyone rule', async () => {
+        // scenario: Alice writes an image to Caroline's DWN with Alice herself as recipient,
```

Comment:
For my understanding: "with Alice herself as recipient": I am not able to understand the scenario, is this common? Is there a more realistic scenario that we can use?